src/libais/ais8_367.cpp

```diff
@@ -163,4 +165,51 @@ Ais8_367_22::Ais8_367_22(const char *nmea_payload, const size_t pad)
   status = AIS_OK;
 }
 
+Ais8_367_22::~Ais8_367_22() {
```

Comment:
I don't think I added this code, it looks like it came in from somebody else's changes that were put into this branch.

Reply:
I've removed this code.